### ISSUE TITLE
go-inovation crashes

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#658]
```
08-05 22:54:14.810 17959-0/com.hajimehoshi.goinovation E/Go: panic: devicescale: JVM is not available yet: is this called from init funcitons?
08-05 22:54:14.812 17959-17959/com.hajimehoshi.goinovation I/Choreographer: Skipped 56 frames!  The application may be doing too much work on its main thread.
08-05 22:54:14.815 17959-0/com.hajimehoshi.goinovation E/Go: goroutine 26 [running]:
    github.com/hajimehoshi/ebiten/internal/devicescale.impl(0x4, 0xa45b1584)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/internal/devicescale/impl_android.go:68 +0x164
    github.com/hajimehoshi/ebiten/internal/devicescale.DeviceScale(0x0, 0x0)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/internal/devicescale/devicescale.go:30 +0x90
    github.com/hajimehoshi/ebiten/internal/ui.(*userInterface).updateGraphicsContext(0xa484b8b8, 0xa45b35b8, 0x58d66000)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/internal/ui/ui_mobile.go:175 +0x68
    github.com/hajimehoshi/ebiten/internal/ui.Run(0x140, 0xf0, 0x0, 0x3ff80000, 0xa44a4601, 0x1d, 0xa45b35b8, 0x58d66000, 0xa4353900, 0xa4838738, ...)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/internal/ui/ui_mobile.go:146 +0x10c
    github.com/hajimehoshi/ebiten.run(0x140, 0xf0, 0x0, 0x3ff80000, 0xa44a4601, 0x1d, 0x58d66000, 0x0, 0x0, 0x0)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/run.go:98 +0x58
08-05 22:54:14.816 17959-0/com.hajimehoshi.goinovation E/Go: github.com/hajimehoshi/ebiten.RunWithoutMainLoop.func1(0x508066c0, 0x507b4110, 0x140, 0xf0, 0x0, 0x3ff80000, 0xa44a4601, 0x1d)
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/run.go:308 +0xc8
    created by github.com/hajimehoshi/ebiten.RunWithoutMainLoop
    	/Users/hajimehoshi/go/src/github.com/hajimehoshi/ebiten/run.go:303 +0xd8
08-05 22:54:14.816 17959-17999/com.hajimehoshi.goinovation A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 17999 (shi.goinovation)

```

**Comment by hajimehoshi** [comment]
https://github.com/golang/mobile/commit/56e3592fa7a0582e84d783579d2eb8e6114e0312 Ooops

**Comment by hajimehoshi** [comment]
Asking https://github.com/golang/go/issues/26815

Until this is solved, let's use gomobile@6621de06e1e9d52c38a8c542770bdbeac9ecfef6

**Comment by hajimehoshi** [comment]
Resolved by https://github.com/hajimehoshi/ebiten/commit/ed99719fdb0e7376c7a6ff0d52c83721c95bebf5
